signimage/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM registry.fedoraproject.org/fedora as ksource
+RUN yum install -y kernel-devel
+RUN cp /usr/src/kernels/*/scripts/sign-file /sign-file
```

Comment:
This layer seems redundant.

In addition to that, are you assuming anything regarding `/usr/src/kernels/*/scripts/sign-file`?
What happens if there are multiple kernels? are we going to copy multiple files? (When I run that container on my fedora host I don't have any kernel in `/usr/src/kernels/`.

Reply:
Thanks for the explanation.

Regarding the first part, I mean that you have 2 identical layers, so we can combine them (2 layers instead of 3 in total).
You override it so I don't have the original file to see.

Reply:
you're right, that `COPY  cp /usr/src/kernels/*/scripts/sign-file /sign-file` is redundant, I'd not spotted that. I have removed it.